tests/instrumentors/logging_instrumentor_tests.rb

```diff
@@ -26,6 +26,33 @@
     ]
   end
 
+  tests("connection logger with query as hash").returns(true) do
+    Excon.stub({:method => :get}, {body: 'body'}, status: 200})
```

Comment:
It appears there is an extra `}` after the status portion here that is causing a syntax error in tests. Could you fix it please?